File: client/src/app/model/login-register-response.ts
import {StaffRole} from './register-req';

export interface LoginRegisterResponse {
  msg:string;
  status:boolean;
  role:StaffRole;

  // constructor(msg: string, status: boolean, role: StaffRole) {
  //   this.msg = msg;
  //   this.status = status;
  //   this.role = role;
  // }
  //
  // get getMessage(){return this.msg}
  // get getStatus(){return this.status}
  // get getRole(){return this.role}
}

File: server/src/main/java/com/upsintern/emplorium/responsemodel/LoginRegisterResponse.java
package com.upsintern.emplorium.responsemodel;

import com.upsintern.emplorium.entity.Staff;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class LoginRegisterResponse{
    String msg;
    boolean status;
    Staff.StaffRole role;
}
